std_msgs/Header header
float64 state_altitude              # altitude reading
float64 state_altitude_variance     # 0 is interpreted as variance unknown
